plugins/copr.py

```diff
@@ -254,6 +269,15 @@ def _remove_repo(cls, repo_filename):
             raise dnf.exceptions.Error(str(e))
 
     @classmethod
+    def _disable_repo(cls, copr_username, copr_projectname):
+        exit_code = call(["dnf", "config-manager", "--set-disabled",
```

Comment:
This is weird to call DNF plugin from DNF plugin externally. We should add this config manager subfunctionality into core lib and call it ftom Python. Opinions @MichaelMraka ?